src/helpers/handlePRIVMSGMessage.js

```diff
@@ -44,6 +44,10 @@ export default (messageData, connection) => {
 		...args
 	} = mri(argv)
 
+	if (command.toLowerCase().trim() === 'reconnect') {
+		return connection.sendReconnect('RECONNECT')
```

Comment:
One last nitpick: `sendReconnect` doesn't take any arguments, so we can remove the string that gets passed in.